config/root.go

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
nit: the general convention is to name the main file of a package after the package (i.e. this should be `config/config.go`)